### PR TITLE
W 12211477 rpa bot 1.1.6 cv

### PR DESCRIPTION
**Pull request by Cristian-Venticinque**

### CODE REVIEW

**Review by dmerlob — APPROVED** [review]

File: rpa-bot/1.1/modules/ROOT/pages/_partials/hard-soft-requirements.adoc [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+| Software | Required version
+| Microsoft .NET Framework
+ a|
+* 4.5
+* 4.7.2
+| Operating System
+ a|
+Any of the following:
+
+* Windows 8.1 / 10 / 11
```
Comment (justus-schach) [inline rpa-bot/1.1/modules/ROOT/pages/_partials/hard-soft-requirements.adoc:28]: Please remove Windows 8.1, this windows version is not supported anymore.

File: rpa-bot/1.1/modules/ROOT/pages/_partials/hard-soft-requirements.adoc [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+* Windows Server 2016 / 2019 / 2022
+|===
+
+=== Requirements for Secure Session Technology
+
+* Windows logins used for secure sessions have privileges to open a local RDP session on `127.0.0.2`.
+* 64-bit Operating System.
+* Dedicated Windows Server machine with enough resources to execute several Processes simultaneously (2 GB or more RAM per session)
+
```
Comment (justus-schach) [inline rpa-bot/1.1/modules/ROOT/pages/_partials/hard-soft-requirements.adoc:37]: Please add the following hint:
* The RPA Bot service must run under the "Local System account" (setup use this account by default)

File: rpa-bot/1.1/modules/ROOT/pages/configuration.adoc [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+[[secure-session]]
+== Secure Session
+
+Enable the bot to use Secure Sessions by entering credentials the bot uses to login to the computer and to open a hidden session. If you don't add credentials here, the bot can only use a running desktop session.
+
+image:bot-configurator-secure-session.png[The RPA Bot configurator application showing the Secure Session settings panel, 50%, 50%]
+
+* *Default Secure Session User*
++
+If you want the bot to use Secure Sessions, enter the name of the user the bot uses to login to the computer and to run a hidden session. If the environment of the computer the bot is installed on is embedded in a domain and you want to use a local user, write the user name as follows: `.\username`.
```
Comment (justus-schach) [inline rpa-bot/1.1/modules/ROOT/pages/configuration.adoc:64]: Please add an example for Domain users. They are often defined in the form of username@mydomain.com
This is not usable here. So the user must enter such a domain user in this form:  mydomain\username
The TLD (.com) must be omitted.

BTW: In one of the next versions we will show an hint in the dialog if the user tries to use the @ mail format.

**Review by justus-schach — COMMENTED** [review]
Made a few comments...

**Pull request merged**